plugins/UsersManager/lang/en.json

```diff
@@ -8,7 +8,7 @@
         "AnonymousUserHasViewAccess2": "Your analytics reports and your visitors information are publicly viewable.",
         "ApplyToAllWebsites": "Apply to all websites",
         "ChangeAllConfirm": "Are you sure you want to give '%s' access to all the websites?",
-        "ChangePasswordConfirm": "Changing the password will also change the user's token_auth. Do you really want to continue?",
+        "ChangePasswordConfirm": "Changing the password will also change the user's old token_auth. This will only happen once. Do you really want to continue?",
```

Comment:
Actually went with a separate message here. This allowed me to sneak a "may require you to login again" into the message while keeping it safe regarding the translation to other languages.